src/root-config-guide/root-config-guide.component.js

```diff
@@ -0,0 +1,192 @@
+import React, { useContext } from "react";
+import PageHeader from "../shared/page-header.component";
+import { useCss } from "kremling";
+import { LocalStorageContext } from "../shared/use-local-storage-data.hook";
+import { SlackLink } from "../shared/links.component";
+import Code from "../shared/code.component";
+import { sharedDepsImportMap } from "../shared/registered-app.component";
+
+export default function HtmlFile(props) {
+  const scope = useCss(css);
+  const [copying, setCopying] = React.useState(false);
+  React.useEffect(() => {
+    if (copying) {
+      const timeoutId = setTimeout(() => {
+        setCopying(false);
+      }, 500);
+
+      return () => clearTimeout(timeoutId);
+    }
+  }, [copying]);
+  const { application } = useContext(LocalStorageContext);
+  if (!application)
+    return <p>You need to register at least one application!</p>;
+
+  const importMap = {
+    [application.name]: window.importMapOverrides.getOverrideMap().imports[
+      application.name
+    ],
+    ...sharedDepsImportMap(application.sharedDeps),
+  };
+  const code = `
+<script type="systemjs-importmap">
+  ${JSON.stringify(importMap, null, 2)}
+</script>`;
+
+  const registerAppCode = `
+singleSpa.registerApplication(
+  '${application.name}',
+  () => System.import('${application.name}'),
+  location => location.pathname.startsWith('${application.pathPrefix}')
+);
+
+singleSpa.start();
+  `;
+  return (
+    <>
+      <PageHeader title="Root configuration" />
+      <article className="card" {...scope}>
+        <section>
+          It's time to talk about the root-config. In single-spa, there is only
+          one html file for your applications. We call it the "single-spa root
+          config." This is different than normal -- you don't get one HTML file
+          per application, just one for all of them. Only having one html file
+          is actually the origin of the single-spa name (only a "single"
+          single-page-application -- don't think about it too hard{" "}
+          <span role="img" aria-label="Smiley emoji">
+            😀
+          </span>
+          )
+        </section>
+        <section>
+          The single-spa root config is something that we can provide for you
+          entirely. Let's use the create-single-spa again!
+        </section>
+        <h2>Creating a single-spa root config</h2>
+        <div>
+          You'll want to create an{" "}
+          <span className="whole-new-git-repo">entirely new git repo</span> for
+          your single-spa root config, separated from your applications.
+        </div>
+        <div>
+          <Code code={`npx create-single-spa --module-type root-config`} />
+        </div>
+        <p>
+          This command won't give you a running root-config out of the box for
+          our playground demonstration! But, fear not, you will understand
+          everything root-config related now!
+        </p>
+        <p>
+          First, lets talk about the{" "}
```

Comment:
Should either be `first, let's talk about import maps` or `first, let's talk about the import map`

```suggestion
          First, let's talk about {" "}
```

Reply:
WOW just searched `lets vs let's` and I never knew "let's" was a contraction of `let us` ! hahahaha
I thought there were only contraction for `bleus has` `bleus is` to form "bleus's"